internal/service/common.go

```diff
@@ -121,6 +123,40 @@ func ConvertFieldFilterParamsToMap(params []string) (map[string][]string, error)
 	return fieldMap, nil
 }
 
+// validateSortField validates a sort field and returns a slice of SortField and a boolean indicating success.
+func validateSortField(field string) ([]store.SortField, bool) {
```

Comment:
The validation function will apply to the type ListDevicesParams since we're not passing a device kind in the request body. We also make other validations to the requests once they arrive before calling List.
Should we move the function to the validation.go? Should we implement validation for ListDevicesParams?

Reply:
Done, It was removed due to the API changes